sources/engine/Stride.Rendering/Rendering/Utils/Math.sdsl

```diff
@@ -114,4 +114,10 @@ shader Math
         vecProjected.xyz /= vecProjected.w;
         return vecProjected;
     }
+
+    //Exponential damping
+    float ExpDecay(float a, float b, float lambda, float dt)
+    {
+        return a + (b - a) * exp(-lambda * dt);
```

Comment:
The arguments here are flipped compared to the c# counter part, is that on purpose ?

Reply:
You mean `a` and `b`? Totally not! Thanks for pointing out, will fix.